main.c

```diff
@@ -388,7 +391,7 @@ int main(int argc, char *argv[])
 		send_cmd("player-play\n");
 	if (flags[FLAG_PAUSE])
 		send_cmd("player-pause\n");
-	if (flags[FLAG_FILE])
+	if (flags[FLAG_FILE] && play_file)
```

Comment:
`play_file` is assigned at line 318 (`case FLAG_FILE:`), I believe this change is redundant.